data/tlg0086/tlg034/tlg0086.tlg034.digicorpus-grc2.xml

```diff
@@ -0,0 +1,529 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+  <teiHeader xml:lang="eng">
+      <fileDesc>
+         <titleStmt>
+           <title xml:lang="grc">Περὶ ποιητικῆς</title>
+            <author>Aristotle</author>
+            <editor>Immanuel Bekker</editor>
+            <funder>Andrew W. Mellon Foundation</funder>
+         </titleStmt>
+        
+        <publicationStmt>
+          <publisher>Trustees of Tufts University</publisher>
+          <pubPlace>Medford, MA</pubPlace>
+          <authority>Perseus Digital Library Project</authority>
+          <publisher>Harvard University</publisher>
+          <pubPlace>Cambridge, MA</pubPlace>
+          <authority>Digital Corpus for Graeco-Arabic Studies</authority>
+          <date type="release" when="2015" cert="low">Digital Corpus for Graeco-Arabic Studies</date>
+          <idno type="filename">tlg0086.tlg034.digicorpus-grc2.xml</idno>
+          <availability>
+            <licence target="https://creativecommons.org/licenses/by-sa/4.0/">Available under a Creative Commons Attribution-ShareAlike 4.0 International License</licence>
+          </availability>
+        </publicationStmt>
+        
+         <sourceDesc>
+            <biblStruct>
+               <monogr>
+                  <title xml:lang="lat">Aristotelis Opera</title>
+                  <author>Aristotle</author>
+                  <editor>Immanuel Bekker</editor>
+                  <imprint>
+                     <publisher>Oxford University Press</publisher>
+                     <pubPlace>Oxford</pubPlace>
+                     <biblScope unit="volume">11</biblScope>
+                     <biblScope unit="page" from="238" to="276">238-276</biblScope>
+                     <date>1837</date>
+                  </imprint>
+               </monogr>
+              <ref target="https://babel.hathitrust.org/cgi/pt?id=uc1.31158006467772&amp;seq=248">Internet Archive</ref>
```

Comment:
This file was quite confusing and as it is not a Perseus, I didn't want to change the header without permission. 
I do not know if we have anything similar anywhere else? Ideally, it would not be in the Perseus repo but rather OGL.
If you would like to refine it, feel free.  I mainly avoided changing things. I supplemented where appropriate but was mainly hands off.
I am all for edits that are in keeping with current best practices.